docs/devices-api/more-info/apple-health.mdx

```diff
@@ -138,6 +138,12 @@ This will be a daily average of all data types.
 **Step 2:** Once this data is queried we will then listen on an hourly basis for any updates to the Apple Health.
 If there are changes then a query will be sent via webhook with just the data that's been added.
 
+<Info>
+  When listening for hourly changes you'll notice that the metadata contains an `hour` field. This
+  is to signify the hour of the day in which the data is aggregated as shown in the [
+  model](/devices-api/more-info/models#metadata).
+</Info>
```

Comment:
Let's update this to show an example metadata payload & be explicit about what format the `hour` will be in.